qa/L0_backend_python/bls/test.sh

```diff
@@ -79,7 +79,9 @@ mkdir -p models/dlpack_identity/1/
 cp ../../python_models/dlpack_identity/model.py models/dlpack_identity/1/
 cp ../../python_models/dlpack_identity/config.pbtxt models/dlpack_identity
 
-cp -r ${DATADIR}/qa_sequence_implicit_model_repository/onnx_nobatch_sequence_int32/ ./models
+cp -r ${DATADIR}/qa_sequence_implicit_model_repository/onnx_nobatch_sequence_int32/ ./models && \
+    (cd models/onnx_nobatch_sequence_int32 && \
+        sed -i "s/kind: KIND_GPU/kind: KIND_CPU/" config.pbtxt)
```

Comment:
I was wondering why is this change required? Also, can you add a test where you send an inference to an onnx model on GPU and check the output tensor provided by BLS to make sure it is on GPU?

Reply:
> I was wondering why is this change required?

I thought we cannot use numpy for GPU tensors as it shows [here](https://github.com/triton-inference-server/python_backend/blob/2b98842559055b4d2837dd972e2392683c846079/src/pb_tensor.cc#L405), that's why I changed it to KIND_CPU so that we can use `as_numpy` in the [unittest](https://github.com/triton-inference-server/server/blob/main/qa/L0_backend_python/python_unittest.py#L48) script.

> Also, can you add a test where you send an inference to an onnx model on GPU and check the output tensor provided by BLS to make sure it is on GPU?

Thanks for the heads up, I will add a test for checking if the output tensor is on GPU.

Reply:
Update: Just found out that we can use dlpack() to replace numpy. I will modify the test.

Reply:
Reverted the changes on onnx model. Added checks to see if output tensor is on GPU when sending inference to onnx model.